### PR TITLE
display splash screen before redirecting to dashboard

### PR DESCRIPTION
**Pull request by ebrehault**

### CODE REVIEW

File: apps/desktop/src/app/login/login.component.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import 'variables';
+.splash {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 99999;
```
Comment (mpellerin42) [inline apps/desktop/src/app/login/login.component.scss:8]: is this super high z-index really needed?

File: apps/desktop/src/app/login/login.component.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.container {
+  position: relative;
+  top: 10vh;
+  width: 600px;
+  height: 80vh;
+  margin: 0 auto;
+  padding: 62px 100px;
+  border-radius: 8px;
```
Comment (mpellerin42) [inline apps/desktop/src/app/login/login.component.scss:19]: Are we using rhythm in this project?

File: apps/desktop/src/app/login/login.component.scss (hunk 1 above)
Comment (ebrehault) [reply apps/desktop/src/app/login/login.component.scss:8]: that's the value we use for splash screen in our different index.html (that's why there is no SCSS variable for it, but I'll create one)

File: libs/pastanaga/src/lib/styles/partials/_variables.scss [hunk 3]
```diff
@@ -254,7 +259,7 @@ $font-size: (
   // ~36px - big article promo title
   xxd: var(--stf-font-size-xxd),
   // ~42px - big section primi title
-  xxxd: var(--stf-font-size-xxxd) // ~60px - main hero claim,
+  xxxd: var(--stf-font-size-xxxd) // ~60px - main hero claim,,
```
Comment (mpellerin42) [inline libs/pastanaga/src/lib/styles/partials/_variables.scss:262]: not sure those extra comma are needed 😅

**Review by mpellerin42 — APPROVED** [review]
Looks good to me (even though you added some commas in a comments ;)

**Pull request merged**